examples/react/src/stories/index.stories.js

```diff
@@ -17,9 +17,11 @@ import FetchComponent from '../FetchComponent';
 import ZeroHeightWithPadding from '../ZeroHeightWithPadding';
 import Hover from '../Hover';
 
-storiesOf('Welcome', module).lokiSkip('to Storybook', () => (
-  <Welcome showApp={linkTo('Button')} />
-));
+storiesOf('Welcome', module)
+  .lokiSkip('to Storybook', () => <Welcome showApp={linkTo('Button')} />)
```

Comment:
So `lokiSkip` will continue to exist and behave like `add`?

Reply:
Yeah, i'll add warnings for now and remove it some time in the future.